Modern/wrapperPlugins/src/main/java/org/bonej/wrapperPlugins/AnisotropyWrapper.java

```diff
@@ -124,7 +125,7 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 	private static UnaryFunctionOp<Matrix4dc, Optional<Ellipsoid>> quadricToEllipsoidOp;
 	private static UnaryFunctionOp<List<Vector3d>, Matrix4dc> solveQuadricOp;
 	private final Function<Ellipsoid, Double> degreeOfAnisotropy =
-		ellipsoid -> 1.0 - ellipsoid.getA() / ellipsoid.getC();
+			ellipsoid -> 1.0 - (1.0/(ellipsoid.getC() * ellipsoid.getC())) / (1.0/(ellipsoid.getA() * ellipsoid.getA()));
 	@SuppressWarnings("unused")
```

Comment:
Could be simplified to `ellipsoid -> 1.0 - (ellipsoid.getA() * ellipsoid.getA())/(ellipsoid.getC() * ellipsoid.getC());` if you wanted - but maybe the current version makes the radius - eigenvalue relationship clearer?

Reply:
I would rather get the eigenvalues straight from the ellipsoid